pylib/cqlshlib/copyutil.py

```diff
@@ -2101,6 +2101,13 @@ def convert_map(val, ct=cql_type):
             return ImmutableDict(frozenset((convert_mandatory(ct.subtypes[0], v[0]), convert(ct.subtypes[1], v[1]))
                                  for v in [split(split_format_str % vv, sep=sep) for vv in split(val)]))
 
+        def convert_vector(val, ct=cql_type):
+            string_coordinates = val.strip("[]").split(',')
+            if len(string_coordinates) != ct.vector_size:
+                raise ParseError("The length of given vector value '%d' is not equal to the vector size from the type definition '%d'" % (len(string_coordinates), ct.vector_size))
+            float_vector = [float(v) for v in string_coordinates]
```

Comment:
This should work for now but it's worth noting that C* 5.0 apparently supports arbitrary types for vector subtypes and the Python driver will aim to do the same in a future release (see [PYTHON-1369](https://datastax-oss.atlassian.net/browse/PYTHON-1369)).  I don't think that needs to be fixed here (since Astra will only be supproting float types) but it's something to at least mention and think about.